tests/helpers/run-command.js

```diff
@@ -43,18 +44,21 @@ module.exports = function run(/* command, args, options */) {
     options.log(`      Running: ${command} ${args.join(' ')} in: ${process.cwd()}`);
 
     let opts = {};
+    args = [`--unhandled-rejections=strict`, `${command}`].concat(args);
+    command = 'node';
     if (process.platform === 'win32') {
-      args = [`"${command}"`].concat(args);
-      command = 'node';
       opts.windowsVerbatimArguments = true;
       opts.stdio = [null, null, null, 'ipc'];
     }
     if (options.env) {
       opts.env = defaults(options.env, process.env);
     }
 
-    debug.info('command: %s, args: %o', command, args);
+    console.log('runCommand: %s, args: %o', command, args);
```

Comment:
I believe this change will result in things being logged in test output. Is this desirable?

Reply:
No, I don’t think we should change this permanently